src/Workspaces/Core/Desktop/Workspace/SQLite/Interop/SqlConnection.cs

```diff
@@ -30,6 +31,18 @@ internal partial class SqlConnection
         private readonly sqlite3 _handle;
 
         /// <summary>
+        /// Whether or not we were able to successfully instantiate this connection. If we weren't
+        /// then we definitely do not want to report any issues about the connection not being
+        /// closed up as the caller won't have an actual instance to close.
+        /// </summary>
+        private readonly bool _successfullyCreated;
```

Comment:
Seems to be cleaner, and allows for explicit cleanup in case the call to `sqlite3_busy_timeout` fails. The constructor is only used in one place so it's easy to update.